### DISCUSSION TITLE
Can a call API to get content for each slide?

### DISCUSSION THREAD

**Discussion by luc-hn** [slidevjs/slidev#1856] [Q&A]
Hi, I want to call API to get content for each slide, please let me know if sli.dev support my expecation.

**Comment by kermanx** [comment]
May I ask what you want to do with the content of the slides? In different contexts there are different ways.

**Reply by luc-hn** [reply]
Hi, I just want to show text like that
![image](https://github.com/user-attachments/assets/611ba3b6-8373-4b8c-bf49-baf03a7304bd)

**Reply by kermanx** [reply]
https://sli.dev/custom/config-parser may help

**Reply by luc-hn** [reply]
Hi, can I use only vue & html for each slide intead of markdown

**Reply by luc-hn** [reply]
I've tried to use transformRawLines but it's not working
![image](https://github.com/user-attachments/assets/4ea188eb-8b22-42cd-943d-4a76a3506024)
![image](https://github.com/user-attachments/assets/7861b63f-a8cf-494d-a009-0825e03305ae)
